### DISCUSSION TITLE
The Real Truth About DreamBerd

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#469] [Announcements]
[Last week](https://github.com/TodePond/DreamBerd/releases/tag/vNaN.NaN), we revealed the truth about DreamBerd:

> DreamBerd's true purpose is to make fun of JavaScript. Just look at this abomination:
>
> `typeof NaN === "number"`
>
> Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.
>
> _That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is to make fun of people who make fun of JavaScript. Just look at this abomination:

> Just look at this abomination.
>
> `typeof NaN === "number"`
>
> Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.

Yeah, you read that right. They think that 'not a number' is 'not a number'. But 'not a number' is a 'number' as part of the IEFFFF spec. You probably didn't know this.

_That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is to make fun of people who make fun of people who make fun of JavaScript. Just look at this abomination:

> Just look at this abomination:
>
> > Just look at this abomination.
> >
> > `typeof NaN === "number"`
> >
> > Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.
>
> Yeah, you read that right. They think that 'not a number' is 'not a number'. But 'not a number' is a 'number' as part of the IEFFFF spec. You probably didn't know this.

Yeah, you read that right. They think that 'not a number' is a 'number'. But if you look closely at the name of 'not a number', you'll notice that it says 'not a number', not 'a number'. You probably didn't know this.

_That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is to make fun of people who make fun of people who make fun of people who make fun JavaScript. Just look at this abomination:

> Just look at this abomination:
>
> > Just look at this abomination:
> >
> > > Just look at this abomination.
> > >
> > > `typeof NaN === "number"`
> > >
> > > Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.
> >
> > Yeah, you read that right. They think that 'not a number' is 'not a number'. But 'not a number' is a 'number' as part of the IEFFFF spec. You probably didn't know this.
>
> Yeah, you read that right. They think that 'not a number' is a 'number'. But if you look closely at the name of 'not a number', you'll notice that it says 'not a number', not 'a number'. You probably didn't know this.

Yeah, you read that right. They think that 'not a number' is 'not a number' as if any of this matters. When push comes to shove, our reality has no meaning, and there's no point in arguing about anything. You probably didn't know that.

_That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is to make fun of people who make fun of people who make fun of people who make fun of people who make fun of JavaScript. Just look at this abomination:

> Just look at this abomination:
>
> > Just look at this abomination:
> >
> > > Just look at this abomination:
> > >
> > > > Just look at this abomination.
> > > >
> > > > `typeof NaN === "number"`
> > > >
> > > > Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.
> > >
> > > Yeah, you read that right. They think that 'not a number' is 'not a number'. But 'not a number' is a 'number' as part of the IEFFFF spec. You probably didn't know this.
> >
> > Yeah, you read that right. They think that 'not a number' is a 'number'. But if you look closely at the name of 'not a number', you'll notice that it says 'not a number', not 'a number'. You probably didn't know this.
>
> Yeah, you read that right. They think that 'not a number' is 'not a number' as if any of this matters. When push comes to shove, our reality has no meaning, and there's no point in arguing about anything. You probably didn't know that.

Yeah, you read that right. They think that nothing matters. But they could just go outside and _touch grass_. If they went to the gym, or created a child as soon as possible, they'd find meaning in their life. You probably didn't know this.

_That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is to make fun of people who make fun of people who make fun of people who make fun of people who make fun of people who make fun of JavaScript. Just look at this abomination:

> Just look at this abomination:
>
> > Just look at this abomination:
> >
> > > Just look at this abomination:
> > >
> > > > Just look at this abomination:
> > > >
> > > > > Just look at this abomination.
> > > > >
> > > > > `typeof NaN === "number"`
> > > > >
> > > > > Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.
> > > >
> > > > Yeah, you read that right. They think that 'not a number' is 'not a number'. But 'not a number' is a 'number' as part of the IEFFFF spec. You probably didn't know this.
> > >
> > > Yeah, you read that right. They think that 'not a number' is a 'number'. But if you look closely at the name of 'not a number', you'll notice that it says 'not a number', not 'a number'. You probably didn't know this.
> >
> > Yeah, you read that right. They think that 'not a number' is 'not a number' as if any of this matters. When push comes to shove, our reality has no meaning, and there's no point in arguing about anything. You probably didn't know that.
>
> Yeah, you read that right. They think that nothing matters. But they could just go outside and _touch grass_. If they went to the gym, or created a child as soon as possible, they'd find meaning in their life. You probably didn't know this.

Yeah, you read that right. They're trying to make out that they're not a nerd like everyone else reading this. (Also getting weird vibes from that child comment). Really, they shouldn't have kids. They should stay single, live in the dark, and work overtime all day to make their startup a success. Too many people don't want to work nowadays. You probably didn't know this.

_That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is to make fun of everyone. Just look at this abomination:

> Just look at this abomination:
>
> > Just look at this abomination:
> >
> > > Just look at this abomination:
> > >
> > > > Just look at this abomination:
> > > >
> > > > > Just look at this abomination:
> > > > >
> > > > > > Just look at this abomination.
> > > > > >
> > > > > > `typeof NaN === "number"`
> > > > > >
> > > > > > Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.
> > > > >
> > > > > Yeah, you read that right. They think that 'not a number' is 'not a number'. But 'not a number' is a 'number' as part of the IEFFFF spec. You probably didn't know this.
> > > >
> > > > Yeah, you read that right. They think that 'not a number' is a 'number'. But if you look closely at the name of 'not a number', you'll notice that it says 'not a number', not 'a number'. You probably didn't know this.
> > >
> > > Yeah, you read that right. They think that 'not a number' is 'not a number' as if any of this matters. When push comes to shove, our reality has no meaning, and there's no point in arguing about anything. You probably didn't know that.
> >
> > Yeah, you read that right. They think that nothing matters. But they could just go outside and _touch grass_. If they went to the gym, or created a child as soon as possible, they'd find meaning in their life. You probably didn't know this.
>
> Yeah, you read that right. They're trying to make out that they're not a nerd like everyone else reading this. (Also getting weird vibes from that child comment). Really, they shouldn't have kids. They should stay single, live in the dark, and work overtime all day to make their startup a success. Too many people don't want to work nowadays. You probably didn't know this.

Yeah, you read that right. Both sides are as bad as each other. But not me. I'm above all that. You probably didn't know this.

_That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is to make fun of people who make fun of everyone. Just look at this abomination:

> Just look at this abomination:
>
> > Just look at this abomination:
> >
> > > Just look at this abomination:
> > >
> > > > Just look at this abomination:
> > > >
> > > > > Just look at this abomination:
> > > > >
> > > > > > Just look at this abomination:
> > > > > >
> > > > > > > Just look at this abomination.
> > > > > > >
> > > > > > > `typeof NaN === "number"`
> > > > > > >
> > > > > > > Yeah, you read that right. JavaScript thinks that 'not a number' is a 'number'. You probably didn't know this.
> > > > > >
> > > > > > Yeah, you read that right. They think that 'not a number' is 'not a number'. But 'not a number' is a 'number' as part of the IEFFFF spec. You probably didn't know this.
> > > > >
> > > > > Yeah, you read that right. They think that 'not a number' is a 'number'. But if you look closely at the name of 'not a number', you'll notice that it says 'not a number', not 'a number'. You probably didn't know this.
> > > >
> > > > Yeah, you read that right. They think that 'not a number' is 'not a number' as if any of this matters. When push comes to shove, our reality has no meaning, and there's no point in arguing about anything. You probably didn't know that.
> > >
> > > Yeah, you read that right. They think that nothing matters. But they could just go outside and _touch grass_. If they went to the gym, or created a child as soon as possible, they'd find meaning in their life. You probably didn't know this.
> >
> > Yeah, you read that right. They're trying to make out that they're not a nerd like everyone else reading this. (Also getting weird vibes from that child comment). Really, they shouldn't have kids. They should stay single, live in the dark, and work overtime all day to make their startup a success. Too many people don't want to work nowadays. You probably didn't know this.
>
> Yeah, you read that right. Both sides are as bad as each other. But not me. I'm better than that. You probably didn't know this.

Yeah, you read that right. They're making fun of people. I'm above all that - I don't make fun of people! You probably didn't know this.

_That's the kind of high level satire you can expect from DreamBerd._

Except... That's not the truth. Here's the real truth:

## The Real Truth About DreamBerd

DreamBerd's true purpose is...

## The Surreal Truth About DreamBerd

For frog's sake, stop seeing everything through that lens.

[❌🐦](https://www.todepond.com/wikiblogarden/social-media/just-block-them/)

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/vaN.aN'>The Real Truth About DreamBerd</a>.</em>

**Comment by l11111111111111** [comment]
haha

**Reply by NotAShelf** [reply]
I don't think the truth was ever mentioned in this post, accident?

**Comment by swift502** [comment]
![image](https://github.com/TodePond/DreamBerd/assets/24359130/8e49bd5e-37b1-40b7-9678-58dede5fd2a3)

**Reply by PalmDevs** [reply]
This is illegal! GitHub, take this down!!!

**Comment by theRealLoopDev** [comment]
I like cheese

**Reply by skifli** [reply]
![image](https://github.com/TodePond/DreamBerd/assets/121291719/c99b8445-c005-4bc7-ab4d-bce7dbd3de72)

**Comment by mybearworld** [comment]
> For frog's sake, stop seeing everything through that lens.

I'm not sure I know the alternative of using my eye's lense - could you enlighten me on how that works?

**Reply by radeeyate** [reply]
simple, get cataract surgery

**Reply by mybearworld** [reply]
Hm... does this solution scale?

**Reply by Ciubix8513** [reply]
You don't need scale, you have just a single user

**Reply by mybearworld** [reply]
Actually, most people see through there eyes. It's a little known fact, but it's true!

**Reply by Ciubix8513** [reply]
Still just 1 user tho

**Comment by opensofias** [comment]
the best thing to make fun of is fun itself 😌.

**Comment by nizarcan** [comment]
This is absolutely brilliant

**Reply by mks-h** [reply]
In the end, it is a high level satire

**Comment by GoldenStack** [comment]
bingus

**Comment by TopHatTacco** [comment]
You're right, I didn't know that. Thanks for the explanation as I truly thought that dreamberd was making fun of JavaScript and the thought of that made me gain sleep at night. I now sleep 2 hours every day and it's increased my work per work a lot. Thanks!

**Comment by ZanderBrown** [comment]
Oh that penny took far too long to drop
